async/src/main/java/org/odk/collect/async/TaskSpec.kt

```diff
@@ -17,4 +17,8 @@ interface TaskSpec {
      * WorkManager framework
      */
     fun getWorkManagerAdapter(): Class<out WorkerAdapter>
+
+    companion object {
+        const val DATA_PROJECT_ID = "projectId"
```

Comment:
I think this should move back or live in a shared object in `collect_app`. `async` doesn't need to know about Projects, and we very well might add a task that doesn't care about them.

Reply:
Done.